pkg/controller/infinispan/infinispan_controller.go

```diff
@@ -663,6 +663,16 @@ func (r *ReconcileInfinispan) Reconcile(request reconcile.Request) (reconcile.Re
 		// Spec updated - return and requeue
 		return reconcile.Result{Requeue: true}, nil
 	}
+
+	// If x-site enable configure the coordinator pods to be selected by the x-site service
+	if infinispan.HasSites() {
+		found, _ := r.applyLabelsToCoordinatorsPod(podList, infinispan, cluster, reqLogger)
+		if !found {
+			// If a coordinator is not found reque early
+			return reconcile.Result{Requeue: true}, err
```

Comment:
@rigazilla this will return empty err or as previous call skips err get

Reply:
yep thanks. I don't think we need to handle the error outside the `applyLabelsToCoordinatorsPod` atm. I'll change also the function return.

Reply:
Yes, we need to handle this error in case of update problems inside applyLabelsToCoordinatorsPod

Reply:
mmm right, to keep things I would handle errors in the function and use only requeue as recovery strategy